src/DependencyInjection/BestItOnDemandAssetsExtension.php

```diff
@@ -18,13 +20,22 @@ class BestItOnDemandAssetsExtension extends Extension
 {
     /**
      * Loads the bundle config.
+     *
```

Comment:
Or "The description can be as long as you would like and ends when a tag is encountered for the first time." -> https://phpdoc.org/docs/latest/guides/docblocks.html#description